### PR TITLE
 #162527519 Redesign the UI for Authors Haven

### PR DESCRIPTION
**Pull request by wrotich**
#### What does this PR do?
```This PR merges the updated UI design for Authors Haven branch to develop branch ```
#### Description of Task to be completed?
``` redesign the UI for signup page```
``` redesign the UI for login page```
``` redesign the navbar and footer ```
#### How should this be manually tested?
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
 ```cd ah-orcas-frontend```
- Checkout to the this branch
 ```git checkout ch-setup-sass-162527519```
- Install the required dependencies:
```npm install```
- Run the local development server:
```npm start```
- You should be able to view the ah-orcas-frontend-application on your browser ```
#### What are the relevant pivotal tracker stories?
[#162527519](https://www.pivotaltracker.com/story/show/162527519)
#### Screenshots (if appropriate)
screen shot 2018-12-10 at 13 11 40
screen shot 2018-12-10 at 13 11 54

screen shot 2018-12-10 at 15 30 59

#### Checklist:

- [X] My code follows the style guidelines of this project
- [ ] At least 2 people have reviewed my PR
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

File: src/components/style.scss [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+
+  .signin-form {
+    order: 1;
+    margin-right: 0;
+    margin-left: 0;
+    padding: 0 30px; }
+
+  .form-title {
+    text-align: center; } }
+@media screen and (max-width: 400px) {
```
[the remark this answers is not in the document]
Comment (evamaina) [reply src/components/style.scss:284]: Already Resolved

File: src/components/style.scss [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+  .social-login {
+    flex-direction: column;}
+
+  .social-label {
+    margin-right: 0;
```
[the remark this answers is not in the document]
Comment (evamaina) [reply src/components/style.scss:291]: Resolved, but still remains

File: src/components/style.scss [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+    margin-top: 6em;
+    width: calc( 70% - 30px);
+    max-width: 100%; 
+  } }
+@media screen and (min-width: 1024px) {
+  .container {
+    max-width: 1200px;
+    margin-left: 4em;
+    margin-top: 4em; } }
+@media screen and (max-width: 768px) {
```
[the remark this answers is not in the document]
Comment (evamaina) [reply src/components/style.scss:237]: Already resolved

File: src/components/style.scss [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+  }
+
+@media screen and (max-width: 1200px) {
+  .container {
+    margin-left: 4em;
+    margin-top: 6em;
+    width: calc( 70% - 30px);
+    max-width: 100%; 
+  } }
+@media screen and (min-width: 1024px) {
```
[the remark this answers is not in the document]
Comment (evamaina) [reply src/components/style.scss:229]: Already Resolved

**Review by TheSteelGuy — APPROVED** [review]
LGTM

**Review by Eyansky — APPROVED** [review]

**Review by Gidraf — APPROVED** [review]

**Pull request merged**